src/Shared/UnitTests/Vacancies.Client/Application/CommandHandlers/SubmitVacancyCommandHandlerTests.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace Esfa.Recruit.UnitTests.Vacancies.Client.Application.CommandHandlers
 {
+    [Trait("Category", "Unit")]
```

Comment:
When the project is called UnitTests then what is the point of Trait with category "Unit"?

Reply:
Because I want to and this is going to shuffled about a bit soon.